### PR TITLE
[PL-46590] PL 1.22.3 release notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-46590](https://harness.atlassian.net/browse/PL-46590) PL 1.22.x SaaS release notes

https://harness.atlassian.net/wiki/spaces/PD/pages/21618098251/NG+Release+SAAS+January+22nd+2024+for+1.22.x-+Build

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot. 

[PL-46590]: https://harness.atlassian.net/browse/PL-46590?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: release-notes/platform.md [hunk 1]
```diff
@@ -82,6 +82,24 @@ The following deprecated API endpoints are longer supported:
 
 ## January 2024
 
+### Version 1.22.x <!--  January 29, 2024 -->
```
Comment (brian-f-harness) [inline release-notes/platform.md:85]:
```suggestion
### Version 1.22.3 <!--  January 29, 2024 -->
```

**Comment by brian-f-harness** [comment]
@saifeemustafaqharness remember to add Zendesk ticket number references when they are included.

File: release-notes/platform.md [hunk 2]
```diff
@@ -82,6 +82,24 @@ The following deprecated API endpoints are longer supported:
...
+
+#### Fixed issues
+
+- `K8S_WATCH` PerpetualTasks remained in the TASK_ASSIGNED state despite being assigned to non-existent delegates, now, issue is fixed by implementing a cron job to reset PerpetualTasks associated with invalid delegates, ensuring proper handling of Kubernetes events (PL-43973)
+
+- Running `terraform apply` for an existing SSO-linked User Group previously resulted in an empty user list. This issue has been addressed. Now, when the user group payload is SSO-linked, the existing users are maintained as is, and the users list in the payload is ignored. 
+  - In cases where the existing user group is SSO-linked and needs to be overridden and delinked in update payload, the existing users will be replaced with the users list provided in the payload. (PL-43763)
+
+- The `platform-service` was not publishing the response count metric. This has been resolved, and the `platform-service` will now be consistently publishing the response count metrics. (PL-43123)
+
```
Comment (brian-f-harness) [inline release-notes/platform.md:102]: Added missing Zendesk ticket numbers. Please add them in and consider the additional comments I left for formatting and structure.

**Review by brian-f-harness — CHANGES_REQUESTED** [review]

**Comment by pratmit** [comment]
Is there a preview link for this?

**Review by brian-f-harness — APPROVED** [review]
@saifeemustafaqharness Just one version number placeholder update. Otherwise, looks good. You may want to update the preview based on the new commits. The current preview has placeholder version numbers.

**Review by pratmit — APPROVED** [review]
LGTM

**Pull request merged**